contracts-periphery/.env.example

```
@@ -0,0 +1 @@
+ETH_RPC_URL=
```

Guessing we're forking mainnet, so let's make this self-documenting
```suggestion
ETH_RPC_URL=yourMainnetRpcUrlHere
```